source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCase.cs

```diff
@@ -25,6 +27,11 @@ public ScenarioXUnitTestCase(IMessageSink messageSink, ITestMethod testMethod, s
                 {  "FeatureTitle", new List<string> { featureName } },
                 {  "Description", new List<string> { scenarioName } }
             };
+
+            if (tags != null && tags.Any())
```

Comment:
Will this ever be null? it seems the caller always passes at least an empty list. Maybe we don't need to check here in that case? And if so, also no need for the default null value in the method signature.